pycaret/internal/persistence.py

```diff
@@ -297,39 +297,41 @@ def save_model(
     logger.info("Adding model into prep_pipe")
 
     if use_case == MLUsecase.TIME_SERIES:
-        model_ = deepcopy(model)
```

Comment:
Without the copy, will it not override the original model object when we add the pipeline?

Reply:
I think it should be fine, because we don't modify the model object itself (it will just be referenced in multiple pipelines). I will add a test to make sure, though.